### DISCUSSION TITLE
Replace Search Results File Icon in with Office Fabric UI Icon

### DISCUSSION THREAD

**Discussion by dontthroworanges** [microsoft-search/pnp-modern-search#3966] [Q&A]
Is it possible either through handlebars or some other mechanism to use a static Office Fabric UI icon in place of the File Icon in the search results web part?

**Comment by kasperbolarsen** [comment]
So the same icon for all items?In which layout?

**Reply by dontthroworanges** [reply]
> So the same icon for all items?In which layout?

Correct. As an example, the search results return physical locations (addresses) from a list, so I'd like to use the "MapPin" icon for all these results rather than a file icon. Currently using the Details List layout.

**Reply by kasperbolarsen** [reply]
If it doesn't have to be fancy you can get something like this:
![image](https://github.com/user-attachments/assets/46f8ec70-d6d1-4455-a5ec-1a54e35ba8be)


This goes into the Title handlebar editor:
`<style>
.pnpicon
{
    font-size:30px;
    padding-bottom: 10px;
    margin-bottom: 5px;
    vertical-align: middle;
    display: inline-block;
}
</style>
<pnp-icon class="pnpicon" data-name="MapPinSolid"  aria-hidden="true"  data-theme-variant="{{JSONstringify @root.theme}}"></pnp-icon> 
<a  href="{{slot item @root.slots.PreviewUrl}}" target="_blank" style="color: {{@root.theme.semanticColors.link}}
    ">
	{{slot item @root.slots.Title}}
</a>
`

**Reply by dontthroworanges** [reply]
@kasperbolarsen thanks for the code! In the end I set up a dedicated column using a space character as the title and used the following to insert the icon into each row:
```
<div style="position: relative; width: 100%;">
    <pnp-icon data-name="MapPin" aria-hidden="true" style="font-size: 16px; color: #444444; position: absolute; left: 7px;"> </pnp-icon>
</div>
```
